scripts/x-date-pickers-pro.exports.json

```diff
@@ -106,6 +106,8 @@
   { "name": "enUS", "kind": "Variable" },
   { "name": "esES", "kind": "Variable" },
   { "name": "ExportedDateRangeCalendarProps", "kind": "Interface" },
+  { "name": "ExportedPickersLayoutSlotsComponent", "kind": "Interface" },
```

Comment:
We should probably not export `ExportedPickersLayoutSlotsComponent` and `ExportedPickersLayoutSlotsComponentsProps` (we can remove them from `PickersLayout/index` and adapt our internal imports).